Library/Formula/qxt.rb

```diff
@@ -0,0 +1,239 @@
+class Qxt < Formula
+  desc "LibQxt is an extension library for Qt"
+  homepage "https://bitbucket.org/libqxt/libqxt/wiki/Home"
+
+  stable do
+    url "https://bitbucket.org/libqxt/libqxt/get/v0.6.2.tar.gz"
+    sha256 "582426d6c81958dd1ac5ca4a9d49807e0a2204b7343e7d49c6613bcb148b5cb8"
+    patch :DATA
```

Comment:
It has been submitted upstream.

https://bitbucket.org/libqxt/libqxt-old-hg/issues/50/install-issue-qxt-060-on-mac-os-x-1063

HEAD has the patch applied.